explanation/intro-to/ebpf.md

```diff
@@ -46,17 +45,16 @@ machine-code to optimize the program's execution.
 
 ## eBPF in Ubuntu Server
 
-In Ubuntu, you can leverage tools like the BPF Compiler Collection (BCC) or
-`bpftrace` to identify bottlenecks, investigate performance degradation, trace
-specific function calls, or create custom monitoring tools to collect data on
-specific kernel or user-space processes without disrupting running services.
-
-Since Ubuntu 24.04, `bpftrace` and `bpfcc-tools` (BCC) are available in every
-Ubuntu Server installation by default as part of our efforts to [enhance the
-application developer's and sysadmin's experience in
+Since Ubuntu 24.04, `bpftrace` and `bpfcc-tools` (the BPF Compiler Collection or BCC) are
+available in every Ubuntu Server installation by default as part of our efforts 
+to [enhance the application developer's and sysadmin's experience in
 Ubuntu](https://discourse.ubuntu.com/t/spec-include-performance-tooling-in-ubuntu/43134).
 
-`bpftrace` and `bpfcc-tools` install a range of tools with a variety of
+In Ubuntu, the BCC identifies bottlenecks, investigates performance degradation, traces
```

Comment:
```suggestion
In Ubuntu, you can use tools like the BCC to identify bottlenecks, investigate performance degradation, trace
```

Reply:
The user is the one doing the investigating, BCC is just what they can use to do it :)